.github/workflows/deploy-core-image.yml

```diff
@@ -4,14 +4,18 @@ on:
   workflow_run:
     workflows: ['build-core']
     types: [completed]
+  push:
+    branches:
+      - main
 
 jobs:
   ar-io-core-deployment:
     runs-on: ubuntu-latest
     if: >
-      github.event.workflow_run.conclusion == 'success' &&
-      (github.event.workflow_run.head_branch == 'develop' ||
-       github.event.workflow_run.head_branch == 'main')
+      github.event_name == 'push' ||
```

Comment:
I'd feel better if we included a `github.event.workflow_run.head_branch == 'main'` condition here too.

Reply:
fixed